### ISSUE TITLE
Cannot get started with 2.0

### ISSUE THREAD

**Issue by vekexasia** [i18next/i18next#463]
i tried an almost empty file -> 
```js
const fs   = require('fs');
const i18n = require('i18next');
const {join:pathJoin, basename} = require('path');
var Backend = require('i18next-node-fs-backend');
i18n
    .use(Backend)

    .init({
          backend: {}

},
    function(callback) {
      console.log('loadato i18n');
    });


module.exports = i18n;
```

but i get this NICE looking stack :)
```
(function (exports, require, module, __filename, __dirname) { !function(e){if("object"==typeof exports&&"undefined"!=typeof module)module.exports=e();else if("function"==typeof define&&define.amd)define([],e);else{var t;t="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:this,t.i18next=e()}}(function(){return function e(t,n,r){function o(a,s){if(!n[a]){if(!t[a]){var u="function"==typeof require&&require;if(!s&&u)return u(a,!0);if(i)return i(a,!0);var l=new Error("Cannot find module '"+a+"'");throw l.code="MODULE_NOT_FOUND",l}var c=n[a]={exports:{}};t[a][0].call(c.exports,function(e){var n=t[a][1][e];return o(n?n:e)},c,c.exports,e,t,n,r)}return n[a].exports}for(var i="function"==typeof require&&require,a=0;a<r.length;a++)o(r[a]);return o}({1:[function(e,t,n){"use strict";function r(e){return e&&e.__esModule?e:{"default":e}}function o(e){if(e&&e.__esModule)return e;var t={};if

TypeError: Cannot read property 'indexOf' of undefined
    at t.u.value (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:1:20259)
    at t.u.value (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:1:21247)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:1:4474
    at Array.forEach (native)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:1:4446
    at Array.forEach (native)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:1:4424
    at t.u.value (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:1:4590)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:2:4558
    at t.u.value (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/bin/index.js:1:6755)
```

**Comment by jamuhl** [comment]
try `/lib` when require:

```
const i18n = require('i18nex/libt');
const {join:pathJoin, basename} = require('path');
var Backend = require('i18next-node-fs-backend/lib');
```

will give better stack output as not minified build

**Comment by vekexasia** [comment]
hey @jamuhl thanks for the suggestion

here is the "nice" stack
```
/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/ResourceStore.js:69
      if (lng.indexOf('.') > -1) {
             ^

TypeError: Cannot read property 'indexOf' of undefined
    at ResourceStore.getResource (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/ResourceStore.js:69:14)
    at ResourceStore.hasResourceBundle (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/ResourceStore.js:141:19)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/BackendConnector.js:182:34
    at Array.forEach (native)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/BackendConnector.js:180:26
    at Array.forEach (native)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/BackendConnector.js:179:23
    at Connector.load (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/BackendConnector.js:190:13)
    at /home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/i18next.js:214:46
    at Connector.load (/home/andrea/git/ristoranti_node/lib/generics/i18n/node_modules/i18next/lib/CacheConnector.js:56:43)

```

**Comment by jamuhl** [comment]
ok found it https://github.com/i18next/i18next/blob/master/src/Translator.js#L184 does not take fallbackLng if no language is set. Will fix this asap.

For now just init with lng:

```
i18n
    .use(Backend)

    .init({
          lng: 'en',
          backend: {}
},
```

**Comment by jamuhl** [comment]
should be fixed in 2.0.1

**Comment by jamuhl** [comment]
feel free to reopen if not fixed
